From 7a22c3be4bf592ca5a8fd33c08effc1b4afc2d1b Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@st.com>
Date: Tue, 26 Apr 2022 11:28:30 +0200
Subject: [PATCH 0370/1695] plat-stm32mp2: conf: support TAMP driver

Default enable TAMP driver for platform stm32mp2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 5e43394ac61631a70092398a543d1a5f9b007f98 plat-stm32mp2: conf: support TAMP driver]
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
Change-Id: Ieefe262a946eb1246270ec29f6eef4917a5734ad
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/249368
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349417
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index c780cc67f..34a2b11e3 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -77,6 +77,7 @@ CFG_STM32_RISAF ?= y
 CFG_STM32_RTC ?= y
 CFG_STM32_SERC ?= y
 CFG_STM32_STGEN ?= y
+CFG_STM32_TAMP ?= y
 CFG_STM32_UART ?= y
 
 # Default enable some test facitilites
-- 
2.47.2

